### PR TITLE
[StaticWeb] Generate StaticWeb resource.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Generate StaticWeb resource.

### CODE REVIEW

File: src/Websites/Websites.Autorest/custom/New-AzStaticWebApp.ps1 [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Runtime.Info(PossibleTypes=([Microsoft.Azure.PowerShell.Cmdlets.Websites.Models.Api20201201.IResourceTags]))]
+    [System.Collections.Hashtable]
+    # Resource tags.
+    ${Tag},
+
+    [Parameter()]
+    [Microsoft.Azure.PowerShell.Cmdlets.Websites.Category('Body')]
+    [System.String]
+    # Description of the newly generated repository.
+    ${TemplatePropertyDescription},
```
Comment (isra-fel) [inline src/Websites/Websites.Autorest/custom/New-AzStaticWebApp.ps1:260]: We should rename these parameters about template:
| old                                     | new                         |
| --------------------------------------- | --------------------------- |
| `TemplatePropertyDescription`           | `ForkRepositoryDescription` |
| `TemplatePropertyIsPrivate`             | `ForkRepositoryIsPrivate`   |
| `TemplatePropertyOwner`                 | `ForkRepositoryOwner`       |
| `TemplatePropertyRepositoryName`        | `ForkRepositoryName`        |
| `TemplatePropertyTemplateRepositoryUrl` | `TemplateRepositoryUrl`     |

**Review by isra-fel — CHANGES_REQUESTED** [review]

File: src/Websites/Websites.Autorest/custom/New-AzStaticWebApp.ps1 (hunk 1 above)
Comment (LucasYao93) [reply src/Websites/Websites.Autorest/custom/New-AzStaticWebApp.ps1]: Done

**Review by isra-fel — APPROVED** [review]
Looks great. Thanks

**Pull request merged**